client/log_client.go

```diff
@@ -399,10 +399,15 @@ func (c *LogClient) AddSequencedLeaves(ctx context.Context, dataByIndex map[int6
 		leaf.LeafIndex = index
 		leaves = append(leaves, leaf)
 	}
-	_, err := c.client.AddSequencedLeaves(ctx, &trillian.AddSequencedLeavesRequest{
+	resp, err := c.client.AddSequencedLeaves(ctx, &trillian.AddSequencedLeavesRequest{
 		LogId:  c.LogID,
 		Leaves: leaves,
 	})
+	for _, leaf := range resp.GetResults() {
```

Comment:
It looks like the client doesn't get to find out which leaves already exist. Maybe that doesn't matter.